slack_cli_hooks/hooks/doctor.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python
+import json
+import platform
+
+from slack_cli_hooks.protocol import (
+    Protocol,
+    build_protocol,
+)
+
+PROTOCOL: Protocol
+
+
+doctor_payload = {
+    "versions": [
+        {
+            "name": "python",
+            "current": platform.python_build()[0],
```

Comment:
Could we use the [python_version()](https://docs.python.org/3.9/library/platform.html#platform.python_version) function, it seems simpler
```suggestion
            "current": platform.python_version(),
```

Reply:
The `python_build()[0]` also includes the `python_revision()` which might be useful? I think it looks cool TBH but open to swapping. As is, here's how this might change:

```diff
-        python: v3.12.0:0fb18b02c8
+        python: 3.12.0
         implementation: CPython
         compiler: Clang 13.0.0 (clang-1300.0.29.30)
```

We could also include a prefixed `v` with `python_branch()` but totally your call!

Reply:
I think the output of `python_version()` is less ambiguous

Lets say I'm a new dev and I install python3 and then try to use the CLI but for some reason my default python path maps to python2, I think an output of `python: 2.7.1` will be easier to understand then `python: v3.7.1:0fb18b02c8`

Reply:
Agree that `2.7.1` easier to understand but I'll poke back and say the longer version is less ambiguous 😜  For debugging having more information might be helpful, but I don't think we'd need to dive into the specific build version. Or at least I hope not...

I'll make this update but if you want to revert back to build information let me know!